beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -714,3 +714,61 @@ func (f *ForkChoice) ParentRoot(root [32]byte) ([32]byte, error) {
 	}
 	return n.parent.root, nil
 }
+
+// OnPayloadAttestationMessage processes a new aggregated payload attestation message.
+func (s *Store) OnPayloadAttestationMessage(
```

Comment:
It would make more sense at least to me if its name was PayloadAttestationAggregation

Reply:
Hey @Redidacove, I still have to add more code in this function

Reply:
I changed the name of the function